eahub/localgroups/views.py

```diff
@@ -107,12 +109,17 @@ def form_valid(self, form):
         recipient = self.profile()
         sender_name = form.cleaned_data["your_name"]
         sender_email_address = form.cleaned_data["your_email_address"]
+        env = environ.Env()
+        admin_email = list(env.dict("ADMINS").values())[0]
+        feedback_url = get_feedback_url(self.request)
```

Comment:
could settings.ADMINS work? I'm asking since in test cases overrides are often utilized, eg [on a parent class](https://github.com/rtcharity/eahub.org/blob/master/eahub/tests/e2e/cases.py), and as I understand the override wouldn't work with the direct envs reading